spec/unit/puppet/provider/apache_directive/augeas_spec.rb

```diff
@@ -5,66 +5,66 @@
 provider_class = Puppet::Type.type(:apache_directive).provider(:augeas)
```

Comment:
Looks like you made this and the other test file executable, which shouldn't be needed.

Reply:
odd, no idea how that happened, fixed it.